rb/spec/integration/selenium/webdriver/driver_spec.rb

```diff
@@ -299,6 +299,22 @@ def save_screenshot_and_assert(path)
           end.to raise_error(Selenium::WebDriver::Error::ScriptTimeoutError)
         end
       end
+
+      describe 'window' do
+        types = ['tab', 'window'] # Stores the valid type values for new_window
+        types.each do |type|
+          it "should be able to open a new #{type}", only: {browser: %i[safari_preview]} do
```

Comment:
The support landed on IE recently (e8d326e2294c7681ba03d274e1bd37f3cc636594), so we should include it too.

Reply:
Nice!

Reply:
Can you please add it? The tests are failing on CI because IE supports this command: https://ci.appveyor.com/project/SeleniumHQ/selenium/builds/23322684/job/myeyect1k0cu57vy#L661.